.github/workflows/test.yaml

```diff
@@ -8,7 +8,11 @@ on:
 
 env:
   PRISMA_TELEMETRY_INFORMATION: 'e2e-tests test.yaml'
+  CI: 1 
   SLACK_WEBHOOK_URL_WORKFLOWS: ${{ secrets.SLACK_WEBHOOK_URL_WORKFLOWS }}
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  SLACK_WEBHOOK_URL_FAILING: ${{ secrets.SLACK_WEBHOOK_URL_FAILING }}
+  DATABASE_URL_POSTGRES_TEMPLATE: ${{ secrets.DATABASE_URL_POSTGRES_TEMPLATE }}
```

Comment:
putting this on the workflow level makes it unnecessary to repeat it for each job